src/engine/test/source/hlp/hlp_test.cpp

```diff
@@ -120,6 +120,26 @@ TEST(hlpTests_logQL, options_parsing)
     ASSERT_EQ("temp", result["_temp"]);
     ASSERT_EQ("temp1", result["_temp1"]);
     ASSERT_EQ("temp2", result["_temp2"]);
+
+}
+
+TEST(hlpTests, annon_type)
+{
+    const char *logQl = "this has an annon ip <_tempIp/ip> in it";
+    const char *event = "this has an annon ip 127.0.0.1 in it";
+
+    auto parseOp = getParserOp(logQl);
+    auto result = parseOp(event);
+
+    ASSERT_EQ("127.0.0.1", result["_tempIp"]);
+
+    const char *logQl2 = "the temp param has an [<_temp/url>] type";
+    const char *event2 = "the temp param has an [incorrect] type";
```

Comment:
We should add this as an alternative of **url_parsing**.
We can rename **url_parsing** to **url_success** and create this test like **url_wrong_format**

Reply:
done